css/src/components/notification.scss

```diff
@@ -33,12 +33,17 @@ $notification-dismiss-margin: $spacer-2 !default;
 		$base: nth($color-set, $color-index-base);
 		$dark: nth($color-set, $color-index-dark);
 		$background: nth($color-set, $color-index-background);
+		$background-hover: nth($color-set, $color-index-background-hover);
 
 		&.notification-#{$name} {
 			border-color: $dark;
 			background-color: $background;
 			color: $dark;
 
+			@if $name == 'success' {
```

Comment:
This is a temporary rule to prevent the success changes from breaking other colored notifications. Fluent's message bar border color is a shade of green that isn't used for any other component and doesn't fit any of the existing success theme variables. Danger and warning have the same issue. It may make sense to add a new variable `success-border` to the theme map, but it means expanding the theme map even more.

https://react.fluentui.dev/?path=/docs/components-messagebar--docs

Reply:
Leaving as `$dark` for now